app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -615,7 +658,13 @@ class Result extends React.Component {
             <div id='drag'/>
             <div id='right-pane'>
               <RightPane
-                {...this.props}
+                {...{
```

Comment:
Haha, well with this change you should just do

```
<RightPane
  assignment_id={this.state.assignment_id}
  ...
```

(No need for the spread operator any more.)